.travis.yml

```diff
@@ -48,7 +48,7 @@ matrix:
     - php: 5.3
       env: PHPCS_BRANCH=2.9.0 SHORT_OPEN_TAGS=true
       dist: precise
-    # Run custom integration test. I just picked to it agains PHP 7.0.
+    # Run custom integration test. I just picked to it against PHP 7.0.
```

Comment:
`I just picked to it against PHP 7.0.` Looks like this is missing the word `run`.

Also, you may as well run PHPCS against the highest PHP version available as that will give the best results if/when new tokens are introduced by PHP itself.